apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/AplDbVersion.java

```diff
@@ -782,7 +782,10 @@ protected int update(int nextUpdate) {
             case 296:
                 apply("ALTER TABLE shard ADD COLUMN IF NOT EXISTS generator_ids ARRAY DEFAULT NULL");
             case 297:
-                return 297;
+                apply("CREATE TABLE IF NOT EXISTS dex_contract (db_id IDENTITY NOT NULL, offer_id BIGINT NOT NULL, " +
+                        "counter_offer_id BIGINT NOT NULL, secret_hash CHAR(64) NOT NULL, height INT NOT NULL, latest BOOLEAN NOT NULL DEFAULT TRUE)");
```

Comment:
Pls assign 'DEFAULT VALUE' for NOT NULL columns if it's possible

Reply:
It's not impossible for this fields.